### PR TITLE
homework1

### PR DESCRIPTION
**Pull request by Zhiqing-Zhu**

### CODE REVIEW

**Comment by iphysresearch** [comment]
Good Job!!!   @Zhiqing-Zhu
---

**Comment by iphysresearch** [comment]
> ## Good Job!!! @Zhiqing-Zhu

@Zhiqing-Zhu 
你需要update branch一下哈

**Comment by iphysresearch** [comment]
Good Job!!!   @Zhiqing-Zhu
---
Python homework:
- Total questions: 108
- Correct answers: 105
- Score: 97.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Zhiqing-Zhu
---
Python homework:
- Total questions: 108
- Correct answers: 105
- Score: 97.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

**Comment by iphysresearch** [comment]
Good Job!!!   @Zhiqing-Zhu
---
Python homework:
- Total questions: 108
- Correct answers: 105
- Score: 97.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 11
- Score: 91.00%
---

File: 2023/homework/zhu_zhiqing/homework-matplotlib_seaborn.ipynb
Comment (iphysresearch) [inline 2023/homework/zhu_zhiqing/homework-matplotlib_seaborn.ipynb:1]: Matplotlib + Seaborn 作业！

File: 2023/homework/zhu_zhiqing/homework_credit_scoring.ipynb
Comment (iphysresearch) [inline 2023/homework/zhu_zhiqing/homework_credit_scoring.ipynb:1]: Sklearn 模型搭建作业！

**Comment by iphysresearch** [comment]
@Zhiqing-Zhu 
Python基础作业和扩展作业在加把油哈！

**Comment by Zhiqing-Zhu** [comment]
2024-01-04 00-43-39 的屏幕截图
2024-01-04 00-48-40 的屏幕截图
2024-01-04 00-57-17 的屏幕截图
2024-01-04 01-06-37 的屏幕截图
2024-01-04 01-07-30 的屏幕截图

**Comment by iphysresearch** [comment]
@Zhiqing-Zhu 
Python基础作业再加加油哈！

**Pull request closed without merging**